best-practices/tech_lead.md

```diff
@@ -0,0 +1,367 @@
+  - does the board reflect the sprint goals?
+- are the tickets in the ready column truly actionable?
+  - try to put on a another developer's mindset and read each ticket in the ready column ... is it actionable for a developer that isn't you?
+- sprint planning meetings should maximize team involvement in vetting sprint goals and tickets
+  - spend 10 minutes thinking about how to make the sprint planning meeting crisp and effective
+  - set up agenda for planning meeting
+  - ensure notes are taken (esp for discussions and action items)
+  - is there a boring part of the meeting you can cut out?
+  - are there conversations among a smaller group that can happen ahead of time, or at another time?
+  - feel free to ask your colleagues for input on how to maximize meeting value
+  - be clear about expectations and outcomes for the planning meeting
+  - start the meeting by stating the meeting objectives.
+
+__During Sprint__
+- run standup meetings (keep on track, crisp, discussions at end)
+- ensure there are ALWAYS prioritized, actionable tickets in the ready column
+- PR review happens promptly.  (within hours;  certainly no more than one day for review)
+- PO and Designer review of work, when required, happens promptly.
+
+__End of Sprint__
+- prepare demos, if desired
+- send emails out communicating progress and achievements (almost always the PO)
+- pick the format/agenda for each retro meeting (could be one format for all).  Be clear about expectations and outcomes.
+- start the meeting by stating the meeting objectives.
+- run the retro meeting (keep on track, crisp, have discussions at end, note action items)
+  - spend 10 minutes ahead of time thinking about how to make the retro meeting crisp and effective
+  - set up the retro agenda
+  - ensure notes are taken, esp for discussions and action items (but also for individual reports)
+  - is there a boring part of the meeting you can cut out?
+  - feel free to ask your colleagues for input (e.g. Josh Greben can rattle of a lot of retro format ideas)
+
+
+# What Does a TL _DO_?
+
+## Pairing
+### With Devs
+- TL should be as available as possible to devs to answer questions or pair
+  - (this is one of the reasons why the TL should not take blocking tickets or large tickets -- the TL priority is to keep the other devs unblocked)
+- If you believe a dev took a ticket but might unknowingly be missing some context or understanding, reach out.
+
+### With PO or Designer
+- Exchange info with PO or Designer - ask questions to clarify work in tickets, or relative priorities, or whatever.  
+
+## Board Grooming
+This is a big part of being tech lead.  The board is our way of knowing the state of work, our way to determine devs who might benefit from help without realizing they need it (e.g. "Naomi took that ticket but I need to make sure she knows about XXX"), our way to ensure there is always prioritized work available when a developer is looking for a task.
```

Comment:
In the scrum methodology:
> The Product Owner is responsible for the Product Backlog, including its content, availability, and ordering.

and the whole team is tasked with refining the items:
> Product Backlog refinement is the act of adding detail, estimates, and order to items in the Product Backlog. This is an ongoing process in which the Product Owner and the Development Team collaborate on the details of Product Backlog items. During Product Backlog refinement, items are reviewed and revised.

https://www.scrum.org/resources/what-is-a-product-backlog